modules/opp-architecture-installation.adoc

```diff
@@ -2,14 +2,10 @@
 //
 // * architecture/opp-architecture.adoc
 
-:_module-type: PROCEDURE
+:_content-type: CONCEPT
 [id="opp-architecture-installation_{context}"]
 = Install {product-title} products
 
-To install {product-title}, you must install two products in a specific order. Install {ocp} first. Then, install {rh-rhacm}. It does not matter what order you install the remaining products: {quay}, {rh-storage-essentials-first}, and {acs}. See the following detailed installation information for each product:
+To install {product-title}, you must install {ocp} followed by {rh-rhacm}. Install the additional products by applying {rh-rhacm} policy sets: {quay}, {rh-storage-essentials-first}, and {acs}. 
```

Comment:
```suggestion
To install {product-title}, you must install {ocp} followed by {rh-rhacm}. Install the additional products by applying {rh-rhacm} the following policy sets: {quay}, {rh-storage-essentials-first}, and {acs}. 
```

Reply:
The colon threw me a little bit without some signpost letting me know it was coming.

Reply:
Thanks Joe! The policy set is the ACM policy set that installs the other products.  I will take a look at this sentence construction to see how I can refine it and keep the meaning of the ACM policy set.